website/pages/docs/k8s/operations/connect-ca-provider.mdx

```diff
@@ -0,0 +1,149 @@
+---
+layout: docs
+page_title: Configuring a Connect CA Provider
+sidebar_title: Configuring a Connect CA Provider
+description: Configuring a Connect CA Provider
+---
+
+# Configuring a Connect CA Provider
+
+~> NOTE: Instructions below should only be used for initially bootstrapping a cluster.
+To update Connect CA provider on an existing cluster or to update any properties, such as tokens, of the CA provider,
+please use the [Update CA Configuration Endpoint](/api/connect/ca#update-ca-configuration).
+
+Consul has support for different CA providers to be used with the Consul Service Mesh.
+Please see [Connect Certificate Management](/docs/connect/ca) for the information on the providers
+we currently support.
+
+Generally, to configure a provider via the Consul Helm chart, you need to follow three steps:
+
+1. Create a configuration file containing your provider information.
```

Comment:
```suggestion
1. Generate a vault token
1. Create a configuration file containing your provider information.
```

Reply:
These instructions are generic, so I'd prefer not to have vault-specific info here